.infrastructure/03_s3.tf

```diff
@@ -0,0 +1,26 @@
+data "aws_caller_identity" "current" {}
+
+locals {
+  account_id = data.aws_caller_identity.current.account_id
+}
```

Comment:
What about using a random number instead of the accountId? I know that the accountId *is not* secret information, but by using a random number you can create a backup bucket on a different region using the same prefix. You can do it with the accountId but in this case, we need to add the region to the bucket name.

```
resource "random_integer" "bucket_random_integer" {
  min = 1
  max = 9999
}
```

Reply:
ok, I used random_integer with the latest commit